src/Client.php

```diff
@@ -4,23 +4,32 @@
 
 use Ginger\ApiClient;
 use Ginger\Ginger;
+use GingerPluginSdk\Helpers\OrderBuilderTrait;
 use GingerPluginSdk\Properties\ClientOptions;
 
+require_once "./vendor/autoload.php";
```

Comment:
Autoloading should be arranged by the consumer of the library

Reply:
Ok.